### PR TITLE
Fix modified indicator

### PR DESCRIPTION
**Pull request by lgeiger**
This addresses part of #1165

But it still fails in a interesting way:
modified

### CODE REVIEW

File: src/notebook/native-window.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function createTitleFeed(state$) {
   const modified$ = state$
     .map(state => state.document.get('notebook'))
     .scan((prev, notebook) => ({
-      modified: prev.notebook === notebook,
+      modified: prev.notebook !== notebook,
```
Comment (lgeiger) [inline src/notebook/native-window.js]:
``` javascript
Immutable.is(prev.notebook, notebook) === false
```

will behave the same.

**Comment by rgbkrk** [comment]
I just realized... We're missing logic for saved. All we know is if it was modified from last time.

The logic should be

``` js
  modified: saved.notebook !== current.notebook
```

Yet we don't have `saved.notebook` available here.

**Comment by jdetle** [comment]
Is this still WIP?

**Comment by rgbkrk** [comment]
It fixes a bug yet doesn't fix the modified feature. It can be merged, we've just got some work ahead of us. I think I know what to do for a follow up PR.

**Comment by ivanov** [comment]
Merging, thank you!

**Pull request merged**